app/javascript/src/components/Dashboard/Contacts/Table/ContactDropdown.jsx

```diff
@@ -1,15 +1,18 @@
 import React from "react";
 
+import { MenuHorizontal } from "neetoicons";
 import { Dropdown } from "neetoui";
 
-import DropdownMenu from "./DropdownMenu";
-
 const ContactDropdown = ({ handleDelete }) => {
   const { Menu, MenuItem } = Dropdown;
+  const renderDropdownMenu = () => <MenuHorizontal size={16} />;
+
   return (
-    <Dropdown customTarget={DropdownMenu}>
+    <Dropdown customTarget={renderDropdownMenu}>
       <Menu>
-        <MenuItem.Button onClick={handleDelete}>Delete</MenuItem.Button>
+        <MenuItem.Button style="danger" onClick={handleDelete}>
+          Delete
+        </MenuItem.Button>
       </Menu>
     </Dropdown>
```

Comment:
Is there no option to Edit the contact? If it's not mentioned in the Figma design, then no need. Just checking.

Reply:
Sorry Yedhin, unfortunately, I missed the reviews for this PR since I was working on the scribble project. Yes, there was no explicit option specified to edit the contact. If I were to include it, I would have to add a button but with no functionality. I thought that would be useless and hence proceeded to remove it.